ceph/init.sls

```diff
@@ -10,6 +10,10 @@ install_ceph_pkgs:
   pkg.installed:
     - pkgs: {{ settings.packages }}
 
+create_ceph_config_file:
+  file.touch:
```

Comment:
shouldn't this be file.managed otherwise it will update the mtime each time this runs:

https://docs.saltstack.com/en/latest/ref/states/all/salt.states.file.html#salt.states.file.touch